core/src/main/scala/akka/kafka/internal/TransactionalProducerStage.scala

```diff
@@ -117,27 +121,32 @@ private final class TransactionalProducerStageLogic[K, V, P](
   private val messageDrainInterval = 10.milliseconds
   private var batchOffsets = TransactionBatch.empty
   private var demandSuspended = false
-  private var commitInProgress = false
+  private var latestSeenConsumerGroupMetadata: ConsumerGroupMetadata = null
 
   private var firstMessage: Option[Envelope[K, V, P]] = None
-
-  private val commitTransactionCB: Try[CommitTransaction] => Unit =
-    createAsyncCallback[Try[CommitTransaction]](commitTransaction _).invoke _
+  private var waitingForLastCommitAck = false
```

Comment:
I can't see any change to this field.

Reply:
Ah, leftover that didn’t end up being needed, I’ll remove

Reply:
Remove in [97ed384](https://github.com/akka/alpakka-kafka/pull/1739/commits/97ed38492f2f8504ad1dd81fa5b5c167cf381c01)